maven-install.sh

```diff
@@ -0,0 +1,2 @@
+#group is overiden to support forked repositories
+./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
```

Comment:
done

On Fri, Nov 20, 2015 at 7:08 PM, Yinan Li <notifications@github.com> wrote:

> In maven-install.sh
> <https://github.com/linkedin/gobblin/pull/475#discussion_r45498650>:
>
> > @@ -0,0 +1,2 @@
> > +#group is overiden to support forked repositories
> > +./gradlew install -Dorg.gradle.parallel=false -PuseHadoop2 -Pversion=0.6-SNAPSHOT -Pgroup=gobblin
>
> Can you make the version number an input argument?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/gobblin/pull/475/files#r45498650>.
>



-- 
"Computers are useless. They can only give you answers."
- Pablo Picasso -